scrapinghub/client.py

```diff
@@ -457,6 +457,13 @@ def schedule(self, spidername=None, **params):
             raise ValueError('Please provide spidername')
         params['project'] = self.projectid
         params['spider'] = spidername or self.spider.name
+        spider_args = params.pop('spider_args', None)
+        if spider_args:
+            if not isinstance(spider_args, dict):
+                raise ValueError("spider_args should be a dictionary")
+            cleaned_args = {k: v for k, v in spider_args.items()
+                            if k not in params}
+            params.update(cleaned_args)
```

Comment:
@vshlapakov please update docs - spider_args should be the only recommended way to pass spider arguments